MovieApp/MovieDetailsViewController.swift

```diff
@@ -0,0 +1,351 @@
+//
+//  MovieDetailsViewController.swift
+//  MovieApp
+//
+//  Created by Ante Pušić on 30.03.2022.
+//
+
+import Foundation
+import UIKit
+import SnapKit
+
+class MovieDetailsViewController: UIViewController {
+    private var scrollView: UIScrollView!
+    private var contentView: UIView!
+    
+    
+    private var backgroundImageView: UIImageView!
+    
+    private var scoreLabel: UILabel!
+    private var userScoreLabel: UILabel!
+    
+    private var titleLabel: UILabel!
+    private var yearLabel: UILabel!
+    
+    private var dateLabel: UILabel!
+    private var countryLabel: UILabel!
+    
+    private var genreLabel: UILabel!
+    private var durationLabel: UILabel!
+    
+    private var starGraphicView: UIImageView!
+    
+    
```

Comment:
Try to keep the line breaks consistent. Also, you don't need more than 1 line break at a time :)